radar/2023-08-01/bdd.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "BDD with Gherkin, Cucumber, and Cypress"
```

Comment:
I would say that the technique we want to adopt is just BDD. Cypress is already on our radar. I would add Gherkin and Cucumber as a separate blip (or two blips).

Reply:
clear will do